tests/acceptance/bootstrap/FeatureContext.php

```diff
@@ -77,11 +77,13 @@ private function phabricatorLogin()
 		} catch(Behat\Mink\Exception\ElementNotFoundException $e)
 		{
 			// Absence of this button just means that the user has authorized Phragile before.
+			return;
```

Comment:
I find it odd to have stuff in `finally` block but not execute it. But after looking at this method for some minutes it looks like this should be addressed separately.